modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/RPCAttributes.cs

```diff
@@ -2,9 +2,15 @@
 
 namespace Godot
 {
+    /// <summary>
+    /// Determines that the AnyPeerAttribute targets methods.
```

Comment:
This definition seems to be describing the `AttributeUsage` attribute instead of the `AnyPeerAttribute`.